pkg/visibility/server.go

```diff
@@ -79,12 +79,17 @@ func CreateAndStartVisibilityServer(ctx context.Context, kueueMgr *qcache.Manage
 	return nil
 }
 
-func applyVisibilityServerOptions(config *genericapiserver.RecommendedConfig) error {
+func applyVisibilityServerOptions(config *genericapiserver.RecommendedConfig, enableInternalCertManagement bool) error {
 	o := genericoptions.NewRecommendedOptions("", api.Codecs.LegacyCodec(visibilityv1beta1.SchemeGroupVersion))
 	o.Etcd = nil
 	o.SecureServing.BindPort = 8082
-	// The directory where TLS certs will be created
-	o.SecureServing.ServerCert.CertDirectory = certDir
+	if enableInternalCertManagement {
```

Comment:
I don't know if this is needed actually.

Without this change, we don't actually "handle" any certificates.

This internalCertManager means that we use opa cert rotator but I don't see Kueue using that for visibility.

@mimowo @tenzen-y 

What do you think we should do for this?

Reply:
Maybe I could just do:
```
	o.SecureServing.ServerCert.CertDirectory = certDir
	if !enableInternalCertManagement {
		o.SecureServing.ServerCert.CertKey.CertFile = certDir + "/tls.crt"
		o.SecureServing.ServerCert.CertKey.KeyFile = certDir + "/tls.key"
	}
```